docs/index.rst

```diff
@@ -36,7 +36,10 @@ Analysis types
 :doc:`markdown/pipelines/scrnaseq`
    Single-cell RNA sequencing
 
-:doc: `markdown/pipelines/guideseq`
+:doc:`markdown/pipelines/airrflow`
+   BCR / TCR sequencing
+
+:doc:`markdown/pipelines/guideseq`
```

Comment:
Not sure where this part will appear on the page, but should `Sarek` be added in this list as well? There was another PR today adding information on how to use the pipeline.

Reply:
It's already there in the WES / WGS sequencing page